### PR TITLE
[17.0][MIG] hr_timesheet_day_week

### PR DESCRIPTION
**Pull request by IriaAlonso**
@dalonsod

### CODE REVIEW

File: hr_timesheet_day_week/static/description/index.html [hunk 1]
```diff
@@ -0,0 +1,416 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (dalonsod) [inline hr_timesheet_day_week/static/description/index.html]: Remove this line due to well-known Odoo warning during v17 addon installation

**Review by dalonsod — APPROVED** [review]
See only comment, when attended I'll merge it

**Comment by IriaAlonso** [comment]
@dalonsod done

**Comment by dalonsod** [comment]
@Anxo82 migration finished

**Pull request merged**